packages/form-footer/src/FormFooter.tsx

```diff
@@ -119,60 +114,41 @@ export default function FormFooter({
   contentClassName,
   className,
 }: FormFooterProps) {
-  const RenderedPrimaryButton: React.ReactNode = useMemo(() => {
-    if (primaryButton) {
-      if (isPrimaryButtonProps(primaryButton)) {
-        return (
+  return (
+    <footer className={cx(footerStyle, className)}>
+      <div className={cx(contentStyle, contentClassName)}>
+        {backButtonText && (
           <Button
-            variant={primaryButton.variant ?? 'primary'}
-            disabled={primaryButton.disabled}
-            onClick={primaryButton.onClick}
-            type={primaryButton.type}
+            variant="default"
+            onClick={onBackClick}
             className={buttonStyle}
+            leftGlyph={<ChevronLeftIcon />}
           >
-            {primaryButton.text}
+            {backButtonText}
           </Button>
-        );
-      }
-
-      if (isComponentType(primaryButton, 'Button')) {
-        return primaryButton;
-      }
-
-      errorOnce(
-        '`primaryButton` prop in `FormFooter` must be either a `Button` component, or object with at minumum a `text` property',
-      );
-    }
-  }, [primaryButton]);
-
-  return (
-    <footer className={cx(footerStyle, className)}>
-      <div className={cx(contentStyle, contentClassName)}>
-        <span className={flexSpan}>
-          {backButtonText && (
+        )}
+        <div className={flexEndContent}>
+          {errorMessage && (
+            <Banner className={bannerStyle} variant="danger">
+              {errorMessage}
+            </Banner>
+          )}
+          {cancelButtonText && (
```

Comment:
In props the descriptions says, `Text for the cancel button. A cancel button will appear regardless of whether text is defined.` but in this case it won't show up. We should update that if it's not the case anymore.

Reply:
README updated!